packages/bits/src/lib/select-v2/combobox-v2/combobox-v2.component.ts

```diff
@@ -126,26 +126,26 @@ export class ComboboxV2Component extends BaseSelectV2 implements AfterContentIni
         });
     }
 
-    public ngAfterViewInit() {
+    public ngAfterViewInit(): void {
         super.ngAfterViewInit();
         if (!this.multiselect) {
             this.dropdown.hide$.subscribe(() => {
                 const lastSelectedOption = this.getLastSelectedOption();
                 if (lastSelectedOption) {
-                    this.setInputValue(lastSelectedOption.viewValue);
+                    this.setInputValue(lastSelectedOption.value);
```

Comment:
It would be good to add a unit test that's red before this fix and green after the fix.

Reply:
Added the e2e test